tests/integration/test_integratord/test_integration_options/test_shuffle_options.py

```diff
@@ -0,0 +1,248 @@
+
+
+@pytest.mark.tier(level=1)
+@pytest.mark.parametrize('configuration, metadata', zip(t2_configurations, t2_configuration_metadata), ids=t2_case_ids)
+def test_shuffle_options(configuration, metadata, set_wazuh_configuration, truncate_monitored_files,
+                         configure_local_internal_options_module, restart_wazuh_daemon_function,
+                         wait_for_start_module):
+    '''
+    description: Check that when configuring the options tag with differents values, the Shuffle integration works as
+                 expected. The test also checks that when it is supposed to fail, it fails.
+
+    wazuh_min_version: 4.5.0
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration and local_internal_options.
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Check integration is enabled
+            - Check the integration is unable to run when expected
+            - Check the integration sends the message and gets a response when expected
+        - teardown:
+            - Restore configuration
+            - Stop wazuh
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration loaded from `configuration_template`.
+        - metadata:
+            type: dict
+            brief: Test case metadata.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set wazuh configuration.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - configure_local_internal_options_module:
+            type: fixture
+            brief: Configure the local internal options file.
+        - restart_wazuh_daemon_function:
+            type: fixture
+            brief: Restart wazuh daemon before starting a test.
+        - wait_for_start_module:
+            type: fixture
+            brief: Detect the start of the Integratord module in the ossec.log
+
+    assertions:
+        - Verify the integration is enabled
+        - Verify no options JSON file is detected
+        - Verify the integration sends message to the integrated app's server
+        - Verify the response code from the integrated app
+
+
+    input_description:
+        - The `config_shuffle_options.yaml` file provides the module configuration for this test.
+        - The `cases_shuffle_options.yaml` file provides the test cases.
+
+    expected_output:
+        - ".*(Enabling integration for: '{integration}')."
+        - ".*ERROR: Unable to run integration for ({integration}) -> integrations"
+        - '.*OS_IntegratorD.*(JSON file for options  doesn't exist)'
+        - '.*Response received.* [200].*'
+
+    '''
+    wazuh_monitor = FileMonitor(LOG_FILE_PATH)
+
+    # Read Response in ossec.log
```

Comment:
I suggest you delete the comment, or if you add it, do not put the file name, but refer to the file in a general way.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4166/commits/1460bad24bed27ff08df90a97d4af108d93e29ff